File: ubuntu/W6/mario.py
n = ''
while (n.isnumeric() == False or int(n) < 1 or int(n) > 8):
    n = input("Hight: ")
n = int(n)
verificação = 0

for i in range(0, n):
    verificação += 1
    for j in range(n, verificação, -1):
        print(" ", end='')
    for c in range(0, verificação):
        print("#", end='')
    print("  " + "#" * verificação, end='')
    print("")

File: ubuntu/W1/mario.c
#include <stdio.h>
#include <cs50.h>

void piramides(int j);

int main(void)
{
    int n;
    do
    {
        n = get_int("Hight: ");
    }
    while (n < 1 || n > 8);
    piramides(n);
}
// Watch anime! they are wonderfull!!!

void piramides(int j)
{
    int verificação = 0;
    for (int i = 0; i < j; i++)
    {
        verificação ++;
        int t = 0;
        for (t = j; t > verificação; t--)
        {
            printf(" ");
        }
        for (int r = 0; r < (verificação * 2); r++)
        {
            printf("#");
            if (r == verificação - 1)
            {
                printf("  ");
            }
        }
        printf("\n");
    }
}